app/templates/courses/teacher-courses-view.jade

```diff
@@ -111,6 +118,18 @@ mixin course-info(course)
           span(data-i18n="courses.view_guide_online")
           |  &mdash; Python
 
+    if course.id === view.utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE && Object.keys(view.videoLevels || {}).length > 0
+      .row.m-t-3
+        each videoLevel in view.videoLevels
+          .col-sm-4
```

Comment:
Does this assume we have a fixed number of video levels?  Should we be more flexible, or not that assumption with a comment?

Reply:
Yes for now we are assuming that its just the 3 levels with videos hard coded in `utils.coffee`. But the `row` and `col` classes are flexible in a way that even if there would be more than 3, they will wrap over to the next row.